doc/contributing/advocacy-ambasador-program.md

```diff
@@ -39,6 +39,15 @@ a discussion issue with the nomination in
 [nodejs/collaborators](https://github.com/nodejs/collaborators/)
 titled `Nomination X to be an Ambassador`, where X is the name of the person
 being nominated.
+If there is no objection from the TSC within 14 days, the nomination is approved.
```

Comment:
This sentence is kind of unrelated to the commit. Maybe it should be a different PR or different commit.

Reply:
I mean its related to onboarding since it defines after how long it is considered accepted

Reply:
```suggestion
If there is no objection within 14 days, the nomination is approved.
```